packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -277,6 +277,19 @@ class FileCollection extends DocumentCollection {
     return this.createFile(data, { dirId })
   }
 
+  /**
+   * Creates for both directory or file.
+   * - Used by StackLink to support CozyClient.create('io.cozy.files', options)
+   */
+  async create(attributes) {
```

Comment:
Can you add the description of `attributes` ? Also, should we mark `createDirectory` and `createFile` private?

Reply:
Done, and done.